packages/forms/src/UIForm/Widget/condition.js

```diff
@@ -1,18 +1,91 @@
 import get from 'lodash/get';
-import includes from 'lodash/includes';
+import findIndex from 'lodash/findIndex';
 
-function getValue(value, strategy) {
-	if (!strategy) {
+function toNumber(value) {
+	if (typeof value === 'string') {
+		return Number(value);
+	}
+	return value;
+}
+
+function toString(value) {
+	if (!value) {
 		return value;
 	}
-	switch (strategy.toLowerCase()) {
+	if (typeof value === 'number') {
+		return JSON.stringify(value);
+	}
+	if (value.toString) {
+		return value.toString();
+	}
+	return value;
+}
+
+function parseStrategy(strategy) {
+	if (!strategy) {
+		return undefined;
+	}
+
+	const start = strategy.indexOf('(');
+	if (start > 0) {
+		const end = strategy.indexOf(')', start);
+		if (end > 0) {
+			const params = strategy
+				.substring(start + 1, end)
+				.split(';')
+				.map(it => {
+					const sep = it.indexOf('=');
+					if (sep > 0) {
+						return {
+							[it.substring(0, sep).trim()]: it.substring(sep + 1, it.length).trim(),
+						};
+					}
+					return {
+						value: it.trim(),
+					};
+				})
+				.reduce(
+					(a, v) => ({
+						...a,
+						...v,
+					}),
+					{},
+				);
+			return {
+				name: strategy.substring(0, start).toLowerCase(),
+				params,
+			};
+		}
+	}
+	return {
+		name: strategy.toLowerCase(),
+		params: {},
+	};
+}
+
+function toEvaluator(value, strategyConfig) {
+	switch (strategyConfig.name) {
```

Comment:
We could change the switch for a configurable array, with the name as the key, and the evaluator fn as value.
This pluggable way will ease the adding of possible new strategies, remove the need to call manually each evaluator with the args and also to have this file grow over time.

Here it could be like :
```javascript
// [...]/conditions.js
import evaluatorsDefinitions from './evaluators'; //

function getEvaluatorFor(strategyConfig, value) {
    return evaluatorsDefinitions[strategyConfig.name](strategyConfig, value);
}
```

The defintions of evaluators could look like : 
```javascript
// [...]/evaluators/index.js
import containsEvaluator from './contains';

export default {
    'contains': containsEvaluator,
    // Bunch of others evaluators
};
```

An evaluator could look like :
```javascript
// [...]/evaluators/contains.js
export default containsEvaluator(strategyConfig, value) {
   // What actually exists 
   // Signature : return expected => value
}
```